.github/workflows/acms.yml

```
@@ -60,15 +60,15 @@ jobs:
     name: "Verify ${{ matrix.starter-kits }} with demo-content=${{ matrix.demo-content }}, content-model=${{ matrix.content-model }}, dam-integration=${{ matrix.dam-integration }}, gdpr-integration=${{ matrix.gdpr-integration }}, core-version=${{ matrix.core-version }} on acquia/drupal-recommended-project"
     runs-on: ubuntu-latest
     env:
-      demo_content: ${{ matrix.demo-content }}
-      content_model: ${{ matrix.content-model }}
-      dam_integration: ${{ matrix.dam-integration }}
-      CONNECTOR_ID: ${{ secrets.CONNECTOR_ID }}
-      GMAPS_KEY: ${{ secrets.GMAPS_KEY }}
-      SEARCH_UUID: ${{ secrets.SEARCH_UUID }}
-      SITESTUDIO_API_KEY: ${{ secrets.SITESTUDIO_API_KEY }}
-      SITESTUDIO_ORG_KEY: ${{ secrets.SITESTUDIO_ORG_KEY }}
-      gdpr_integration: ${{ matrix.gdpr-integration }}
+      demo-content: ${{ matrix.demo-content }}
+      content-model: ${{ matrix.content-model }}
+      dam-integration: ${{ matrix.dam-integration }}
+      connector-id: ${{ secrets.CONNECTOR_ID }}
+      gmaps-key: ${{ secrets.GMAPS_KEY }}
+      search-uuid: ${{ secrets.SEARCH_UUID }}
+      sitestudio-api-key: ${{ secrets.SITESTUDIO_API_KEY }}
+      sitestudio-org-key: ${{ secrets.SITESTUDIO_ORG_KEY }}
+      gdpr-integration: ${{ matrix.gdpr-integration }}
```

Optimize the 'env' section by excluding non-secret environment variables and providing them via site installation command, [here](https://github.com/acquia/acquia-cms-starterkit/pull/115/files#diff-ab950cee88a3e7ec09914fd1df3f73814be9d101e26941ccbfe03e1de3cecec3R125).
```suggestion
      connector-id: ${{ secrets.CONNECTOR_ID }}
      gmaps-key: ${{ secrets.GMAPS_KEY }}
      search-uuid: ${{ secrets.SEARCH_UUID }}
      sitestudio-api-key: ${{ secrets.SITESTUDIO_API_KEY }}
      sitestudio-org-key: ${{ secrets.SITESTUDIO_ORG_KEY }}
```

---

As discussed with @vishalkhode1 this needs to handle separately.